core/common/assets/js/api/modules/errors/base-error.js

```diff
@@ -0,0 +1,64 @@
+export default class BaseError extends Error {
+	/**
+	 * Status code of the response
+	 *
+	 * @type {number}
+	 */
+	status = 0;
```

Comment:
What is the difference between `status` and `getStatus`?

Reply:
You are right, will remove it.